target/consensource-ui
# Perichsen/typescript services impl

## Proposed change/fix

- Refactor all .js files to .ts
- Tests for services
- Enable ESLint

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- `npm run watch` and verify that all user interactions are still functioning properly
- `npm run test` and verify that all tests are passing
- `npm run lint` and verify there are no errors
